packages/react/src/components/common/country-dial-codes.tsx

```diff
@@ -0,0 +1,208 @@
+export const countryDialCodes = [
```

Comment:
I assume this came from the previous implementation?

Reply:
Could this be moved to `ui` so Angular and Vue can grab it too, without having to copy and paste it into our own implementation?

Reply:
Yes, we should.  Apparently everyone's re-using this handy list:

https://github.com/uber/baseweb/blob/8255ff344606e4f702af13bd479e5a1e1d0e395d/src/phone-input/constants.js#L18-L321

From: https://github.com/jackocnr/intl-tel-input/blob/master/src/js/data.js

Frankly, I think this is actually a better list since it'll allow us to to key off of the `navigator.language` property by default.

Reply:
I just leaned on how `amplify-js` defined country dial codes: https://github.com/aws-amplify/amplify-js/blob/df95ea3724eb6406f64b03f25086cd3e8644cb5f/packages/aws-amplify-react/src/Auth/common/country-dial-codes.tsx

I like the idea of using that list and building our dial codes from that. I'll make a change.